types/asana/index.d.ts

```diff
@@ -1970,6 +1977,22 @@ declare namespace asana {
          * @return {Promise} The requested resource
          */
         findById(section: string | number, params?: Params, dispatchOptions?: any): Promise<Sections.Type>;
+
+        /**
+         * * Adds the specified task to a specific section of project. Returns empty object.
+         *   * @param {String} section The section to add task to.
+         *   * @param {Object} data Data for the request
+         *   * @param {String} data.task The id of task to be added to specified section.
+         *   * @param {String} data.insert_after An existing task within this section after which the added task should be inserted. Cannot be provided together with insert_before.
+         *   * @param {String} data.insert_before An existing task within this section before which the added task should be inserted. Cannot be provided together with insert_after.
+         *   * @param {Object} [dispatchOptions] Options, if any, to pass the dispatcher for the request
+         *   * @return {Promise} The response from the API
+         * @param project
+         * @param data
+         * @param dispatchOptions?
+         * @return
+         */
+        addTask(section: string | number, data: Sections.SectionsParams, dispatchOptions?: any): Promise<{}>;
```

Comment:
Is there a better type for the promise than `{}`?

Reply:
@rbuckton Actually this is what Asana `addTask` resolves to 🤷‍♂

Reply:
The `@param` tags are duplicated and the JSDoc formatting seems to be off, once that's addressed I can merge

Reply:
Thanks @rbuckton and sorry for indentation, did not even notice that 😥